templates/partners/public-cloud.html

```diff
@@ -7,42 +7,54 @@
   </div>
 </section>
 
-<section class="p-strip is-deep">
-  <div class="u-fixed-width">
+<section class="p-strip has-accent-border--top-left is-deep">
+  <div class="u-fixed-width u-sv3">
     <h2>What&rsquo;s included</h2>
     <p>There are many benefits for partners in the Canonical Public Cloud programme:</p>
-    <ul class="p-list is-split u-no-margin--bottom">
-      <li class="p-list__item is-ticked">Ubuntu images including Ubuntu Pro are tuned and tested for each cloud to ensure optimal performance</li>
-      <li class="p-list__item is-ticked">Automated publication across all cloud availability zones for facilitated operations</li>
-      <li class="p-list__item is-ticked">Be featured as a Public Cloud Programme partner on the Canonical website, including the ability to use our trademarks and logos</li>
-      <li class="p-list__item is-ticked">Regular and automated refresh of Ubuntu images for enhanced user experience and increased security</li>
-      <li class="p-list__item is-ticked">Automated publication across all cloud availability zones for more efficient operations</li>
-      <li class="p-list__item is-ticked">Joint marketing opportunities, including revenue-sharing for support, services and management tools delivered under Ubuntu Advantage for Infrastructure</li>
-    </ul>
+  </div>
+  <div class="row">
+    <div class="col-6 p-card--strip-thin">
+      <h3 class="p-heading--4">Certification</h3>
+      <ul class="p-list u-no-margin--bottom">
+        <li class="p-list__item is-ticked">Ubuntu images including Ubuntu Pro are tuned and tested for each cloud to ensure optimal performance</li>
+        <li class="p-list__item is-ticked">Automated publication across all cloud availability zones for facilitated operations</li>
+        <li class="p-list__item is-ticked">Be featured as a Public Cloud Programme partner on the Canonical website, including the ability to use our trademarks and logos</li>
+      </ul>
+    </div>
+    <div class="col-6 p-card--strip-thin">
+      <h3 class="p-heading--4">Sales enablement access</h3>
+      <ul class="p-list u-no-margin--bottom">
+        <li class="p-list__item is-ticked">Regular and automated refresh of Ubuntu images for enhanced user experience and increased security</li>
+        <li class="p-list__item is-ticked">Automated publication across all cloud availability zones for more efficient operations</li>
+        <li class="p-list__item is-ticked">Joint marketing opportunities, including revenue-sharing for support, services and management tools delivered under Ubuntu Advantage for Infrastructure</li>
+      </ul>
+    </div>
   </div>
 </section>
 
-<div class="u-fixed-width">
-  <hr class="p-separator">
-</div>
-
-<section class="p-strip">
+<section class="p-strip has-accent-border--top-right is-deep">
   <div class="u-fixed-width u-align--center">
     <div class="p-logo-section">
       <h4 class="p-muted-heading u-no-max-width">A selection of our public cloud partners</h4>
```

Comment:
Missing logo changes https://docs.google.com/document/d/1O3KUX41hwsehnAedYV5Pp4i_aMrKDTGYiri5uR6N9fc/edit?disco=AAAAJNwxDUM

Reply:
being addressed in this [issue](https://app.zenhub.com/workspaces/-web-tribe-5931746dba512f05402b61f6/issues/canonical-web-and-design/web-squad/5260)